src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -468,6 +469,7 @@ public void testRS256() throws Exception {
         Assert.assertEquals(0, creds.getBackendRoles().size());
     }
 
+    @Ignore
```

Comment:
Before this get merged please create issues and reference in the code for tests that still have @ignore them

Reply:
Will do! This is still a draft as I am working on CI issues. I'm planning to remove this after issues are resolved.

Reply:
This test was fixed in https://github.com/opensearch-project/security/pull/1711. I removed @Ignore to re-enable the test and applied the fix.